mrp_production_byproduct_cost_share/views/mrp_production_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 ForgFlow S.L
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="mrp_production_form_view" model="ir.ui.view">
+        <field name="name">mrp.production.form</field>
+        <field name="model">mrp.production</field>
+        <field name="inherit_id" ref="mrp.mrp_production_form_view" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='move_finished_ids']/tree//field[@name='allowed_operation_ids']"
+                position="after"
+            >
+            <!-- Required for test_fifo_byproduct -->
+                <field name="cost_share" />
```

Comment:
test_fifo_byproduct is mentioned in Odoo S.A.'s commit, but there is no such test here